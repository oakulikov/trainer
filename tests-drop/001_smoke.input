result,oddF,oddX,oddL
L,1.83,3.41,4.23
F,1.83,3.63,4.21
L,1.85,3.38,4.16
F,1.83,3.45,4.47
L,1.85,3.57,4.08
F,1.81,3.79,4.04