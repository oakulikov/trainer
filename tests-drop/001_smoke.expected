event_number,result,oddF,oddX,oddL,betF,betX,betL,lossF,lossX,lossL,total,uf,ux,ul,pattern
6,F,1.81,3.79,4.04,12350,3600,25200,0,13600,101750,60000,0,6,1,GREEN
5,L,1.85,3.57,4.08,38500,12300,19600,71200,43900,0,50000,1,5,0,
4,F,1.83,3.45,4.47,16300,10450,13300,0,32450,59450,40000,0,4,1,
3,L,1.85,3.38,4.16,25750,7150,8250,47600,24050,0,30000,1,3,0,
2,F,1.83,3.63,4.21,18950,5700,6650,0,14900,27950,20000,0,2,1,
1,L,1.83,3.41,4.23,12050,4150,3100,22050,14150,0,10000,1,1,0,